### PR TITLE
Add Rhodium page

### PR DESCRIPTION
**Pull request by vincent-nguyen-sys**
## Description
This serves to a page about Rhodium including general info and fun facts.

## Todos
Notable points that this PR has either accomplished or will accomplish.
  - [ ] Create a page about rhodium facts.

## Questions
- [ ] Are these fun facts sufficient?

## Status
- [ ] Ready to go

### CODE REVIEW

File: elements/rhodium.md [hunk 1]
```diff
@@ -6,10 +6,10 @@ title: Rhodium
 Symbol : Rh
 Atomic Number : 45
 
-Rhodium makes up only 0.00002 ppm of the earth's crust.
+Rhodium makes up only 0.00002 ppm of the earth's crust! 
 
-Fun Facts:
+Other Fun Facts:
```
Comment (javabster) [inline elements/rhodium.md:11]: I think this should just say "fun facts" not "other fun facts"

File: elements/rhodium.md [hunk 2]
```diff
@@ -6,10 +6,10 @@ title: Rhodium
...
 - ~87% of all rhodium produced is used for catalytic converters.
-- The price of an ounce of Rhodium (~$19,000) is over 10x the price of an ounce of gold (~$1,800).
+- The price of an ounce of Rhodium (\~$19,000) is over 10x the price of an ounce of gold (\~$1,800).
 - Rhodium-103 is the only stable isotope of rhodium.
 - Rhodium is a common metal used in jewelry plating, which some refer to as white gold.
```
Comment (javabster) [inline elements/rhodium.md:15]: This is indeed a fun fact!

**Review by javabster — CHANGES_REQUESTED** [review]

File: elements/rhodium.md (hunk 2 above)
Comment (vincent-nguyen-sys) [reply elements/rhodium.md:15]: The strikethrough should be solved also. Thanks!

**Review by javabster — APPROVED** [review]
LGTM 🚀

**Pull request merged**